config/i18n.ts

```diff
@@ -30,6 +30,9 @@ export const countryLocaleVariants: Record<string, (LocaleObjectData & { country
     // { code: 'ar-AE', name: 'Arabic (U.A.E.)' },
     // { code: 'ar-YE', name: 'Arabic (Yemen)' },
   ],
+  ckb: [
```

Comment:
remove this entry

Reply:
I can remove it and add missing entries using `en`, then you can update your local and add the translation for those missing entries

Reply:
update your local and change translatations in the following lines: 54, 430 and 562

Reply:
Added phrases were translated